src/Infrastructure.EntityFramework/Repositories/OrganizationRepository.cs

```diff
@@ -224,4 +224,21 @@ public async Task<SelfHostedOrganizationDetails> GetSelfHostedOrganizationDetail
             return selfHostedOrganization;
         }
     }
+
+    public async Task<IEnumerable<string>> GetOwnerEmailAddressesById(Guid organizationId)
+    {
+        using var scope = ServiceScopeFactory.CreateScope();
+
+        var dbContext = GetDatabaseContext(scope);
+
```

Comment:
Do we need to have .Organizations in the query since OrganizationUsers already have OrganizationId. And we might to check the query because it may return duplicate emails.

See suggested query below

```
var query = (from u in dbContext.Users
                      join ou in dbContext.OrganizationUsers
                      on u.id equals ou.UserId
                      where ou.Type == 0 && ou.Status == 2
                      group u by u.email into grouped
                      select grouped.Key);
```

Reply:
Hey @cyprain-okeke, I've updated the LINQ to SQL in this commit based on your feedback: https://github.com/bitwarden/server/pull/3319/commits/f8354d15b59c6b773ab1848fccb5ecbf7d00755f

I also noticed that I wasn't using the `organizationId` in the WHERE clause, so I added that in.